magicctapipe/utils/functions.py

```diff
@@ -373,7 +397,7 @@ def get_dl2_mean(event_data):
         reco_ra_mean, reco_dec_mean = calculate_mean_direction(
```

Comment:
even while your current change is just reproducing the old behaviour, I have some doubts concerning this part. 
averaging is done first for az/alt direction and here you are doing it again for lon/lat coordinates. 
It is more or less fine, because for the camera size the spherical plane is nearly flat and thus a rotation of the two coordinate systems would end up as a linear transformation, but the spherical shape I think would still produce some small differences (I guess mostly pronounced for large lat) that would end up that reconstructed az/alt direction and lat/lon direction slightly mismatch one another. 
I think it should be simple to just use the coordinate transformation to go from averaged alt/az to lat/lon. This would also make it easier to maintain the function later on. 
As you prefer you might change it immediately here, or keep it as it is and we rediscuss it in a separate issue.